certificate-manager/src/main/java/io/strimzi/certs/OpenSslCertManager.java

```diff
@@ -77,6 +84,84 @@ public void generateSelfSignedCert(File keyFile, File certFile, Subject sbj, int
         }
     }
 
+    @Override
+    public void addCertToTrustStore(File certFile, String certAlias, File trustStoreFile, String trustStorePassword)
+            throws IOException, CertificateException, KeyStoreException, NoSuchAlgorithmException {
+
+        try {
+            FileInputStream isTrustStore = null;
+            try {
+                // check if the truststore file is empty or not, for loading its content eventually
+                if (trustStoreFile.length() > 0) {
+                    isTrustStore = new FileInputStream(trustStoreFile);
```

Comment:
So what if it is empty? Won't that result in some NPEs down the stream? Or does the Keystore class handle it?  Might be worth adding a clarification comment.

Reply:
Yes, it's handled by `KeyStore`. On the `load` method you can pass a `null` input stream so that it opens an "empty" store that then you can fill and save in an output stream using the `store` method. If the input stream is not `null`, the `KeyStore` class is loaded with its content.

Reply:
Ok. As I said, might be worth the comment.